sw/supervision/paparazzicenter.glade

```diff
@@ -1171,6 +1171,7 @@
                                 <property name="can_focus">True</property>
                                 <property name="receives_default">False</property>
                                 <property name="use_action_appearance">False</property>
+                                <accelerator key="c" signal="activate" modifiers="GDK_MOD1_MASK"/>
```

Comment:
it would be better if it was in a separated pull request since not related to the ins